dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ReferenceConfig.java

```diff
@@ -369,19 +373,43 @@ private T createProxy(Map<String, String> map) {
                 List<Invoker<?>> invokers = new ArrayList<Invoker<?>>();
                 URL registryURL = null;
                 for (URL url : urls) {
-                    invokers.add(REF_PROTOCOL.refer(interfaceClass, url));
+                    Invoker<?> referInvoker = REF_PROTOCOL.refer(interfaceClass, url);
```

Comment:
If the provider didn't start first. Wait consumer start for a while, when provider start, the provider still be subscribe?

Reply:
If check=true, an error will be reported, otherwise it will be subscribed